Wikipedia/UI-V5/WMFNearbySectionController.m

```diff
@@ -79,8 +82,8 @@ - (NSString*)footerText {
 }
 
 - (NSArray*)items {
-    if ([self.nearbyResults.results count] > 0) {
-        return self.nearbyResults.results;
+    if ([self.viewModel.locationSearchResults.results count] > 0) {
```

Comment:
Bahh unnecessary indirection. `WMFLocationSearchResults` is immutable, and all you're asking me to do is write tedious proxy getter/setters when there's nothing to hide.